src/Persistence/MassTransit.Azure.Table/Configuration/Saga/ConcurrencyMode.cs

```diff
@@ -0,0 +1,7 @@
+namespace MassTransit.Azure.Table
+{
+    public enum ConcurrencyMode
```

Comment:
If there no way to change it, remove this and just document that it's always optimistic.

Reply:
Removed